### PR TITLE
Installation steps

### PR DESCRIPTION
**Pull request by darshandamre**
created github readme, fixed layout and added installation steps

[Preview](https://component-lib-git-installation-steps-darshandamre.vercel.app/)

### CODE REVIEW

**Review by tewarimohit — APPROVED** [review]
Looks Good to merge...

File: js/docs.js [hunk 1]
```diff
@@ -13,3 +13,15 @@ copyBtnList.forEach(copyBtn => {
     navigator.clipboard.writeText(value);
   });
 });
+
+// scroll behaviour
+const sideBarLinks = document.querySelectorAll(".sidebar a");
```
Comment (tanishk15sharma) [inline js/docs.js]: you can declare sideBarLinks in the beginning

File: pages/docs.html [hunk 2]
```diff
@@ -19,31 +19,29 @@
...
-            <button class="light-mode">
-              <i class="fas fa-sun"></i>
-            </button>
-          </li>
-        </ul>
-      </nav>
+    <header class="header">
+      <div class="header__content width-90 mx-auto py-2">
+        <a href="/" class="logo"
+          ><img src="../assets/52-designs-logo-500.svg" alt="52 designs"
```
Comment (tanishk15sharma) [inline pages/docs.html:25]: you can add "52 designs logo" on alt tag for more specificity

**Review by tanishk15sharma — APPROVED** [review]
your code is clean

File: pages/docs.html (hunk 2 above)
Comment (darshandamre) [reply pages/docs.html:25]: thats too redundant in my opinion

File: js/docs.js (hunk 1 above)
Comment (darshandamre) [reply js/docs.js]: resolved

**Pull request merged**